### PR TITLE
Return total supply in private JSONRPC

### PR DESCRIPTION
**Pull request by ninjaahhh**

### CODE REVIEW

File: quarkchain/cluster/master.py [hunk 1]
```diff
@@ -1647,6 +1647,35 @@ def is_mining(self):
...
+                if "balances" in alloc_data:
+                    balances = alloc_data["balances"]
+                for k, v in balances.items():
+                    ret += v if k == "QKC" else 0
+
+        # return None if stats not ready
+        if len(self.branch_to_shard_stats) != len(m_coinbase):
+            return None
+
+        ret += (
```
Comment (qizhou) [inline quarkchain/cluster/master.py]: Decay is not applied

**Comment by ninjaahhh** [comment]
hmm github PR won't catch up with updated branch

**Comment by ninjaahhh** [comment]
image

https://www.githubstatus.com

no wonder

File: quarkchain/cluster/master.py (hunk 1 above)
Comment (ninjaahhh) [reply quarkchain/cluster/master.py]: done

**Pull request merged**